bnd.bnd

```diff
@@ -0,0 +1,10 @@
+Bundle-Description: HiveMQ Java MQTT Client 
+Bundle-License: Apache 2.0 License
+Export-Package: \
+  com.hivemq.client.mqtt.*, \
+  com.hivemq.client.mqtt.rx.*, \
+  com.hivemq.client.mqtt.rx.reactivestreams.*, \
+  com.hivemq.client.mqtt.util.*
+Import-Package: \
+  *
```

Comment:
We should import the dependencies specifically.
Can the gradle plugin generate this automatically from the dependencies?

Reply:
This can be removed at all, it's a left-over and bnd's standard. bnd only imports needed packages anyway. I tried to exclude some imports (make them optional) before but that did not work (I thought I could make rxjava optional, so you don't have to provide that if you don't use the trx-API, but it's still needed internally.